### ISSUE TITLE
Multiple photos for JEDI Corner

### ISSUE THREAD

**Issue by rycoley** [datascijedi/website#32]
Current JEDI Corner has two authors (Gelman and Sturdevant). This is not uncommon. Is it possible to have a template that allows for multiple photos?

**Comment by fatso-jetson** [comment]
@rycoley It would be possible to come up with a design for multiple individual author photos, but as the number of authors varies from month to month, this might leave the listing page looking a little all over the place. 

I know that Amstat News sometimes runs an illustrative image to accompany JEDI Corners, but we'd want to check the licencing options on those images before reusing to avoid any potential copyright infringements.

There are two options that are more straightforward to implement:

1. When there are multiple authors, we could photoshop the various headshots together into one image file. Headshots do come in different dimensions and quality, though, so this might take a bit of work to get looking nice.
2. We could run a screengrab of the print version of the JEDI Corner article. I've implemented this in 53eb75a so you can see what it looks like for a couple of the listings.

Let me know your preference, or if you'd like to explore alternative options.

**Comment by ravicodelabs** [comment]
Hey @brtarran, chiming in with a few ideas:

1) Some reworking of a listing item's layout would likely be needed for this, but we can use a loop in the `jedi-corner.ejs` template to include multiple images when needed, e.g. then a new version of an item from `jedi-corner.yml` might look like this (note in particular the `images` key:

```
- title: "The Math Alliance"
  author: David Goldberg, Jacqueline Hughes-Oliver, Leslie McClure, and Javier Rojo
  href: https://magazine.amstat.org/blog/2023/09/01/the-math-alliance/
  intro: The Math Alliance is a national mentoring community of faculty (mentors) and students (scholars) focusing on increasing traditionally excluded American minorities in the quantitative sciences professions.
  date: 09/01/2023
  images:
    - image: images/jedi-corner/grad-fair-pic.jpg
      alt: Photo of attendees picking up graduate fair materials at Field of Dreams Conference registration table
    - image: images/jedi-corner/grad-fair-pic-2.jpg
      alt: Another grad-fair-pic
```

2) Relatedly, it seems that issue #109 is caused by the use of float. _In particular, since the image style currently uses `float: left`, the image is taken out of the normal document layout, and so when the text is too short, the line breaks start from the bottom of the text rather than the bottom of the image, and ultimately cause the subsequent listing item to protrude into the current one._ So, maybe reworking the layout without using `float: left` would address issue #109 as well, or using some other workaround for when the text is shorter than the image. 

3) It probably also makes sense to move the `<style>` tag currently in the `jedi-corner.ejs` outside of the loop, as it only needs to be included once. To that note, the style info itself could also be moved out of the template and into `css/styles.css`, which is included in all of the rendered pages anyway. 

Let me know if you have any thoughts/questions!

**Comment by ravicodelabs** [comment]
Update: Issue #109 was fixed in #111. I've also moved the CSS from the template to `css/styles.css` in that merge. So, only the first point of my prior comment remains.

**Comment by ravicodelabs** [comment]
Spoke with @rycoley and workarounds are working fine for this, so closing this issue.
